index.bs

```diff
@@ -651,8 +652,8 @@ agent MUST run the following steps:
 1. Let |dedicated| be the negation of |allowPooling|.
 1. Let |serverCertificateFingerprints| be {{WebTransport/constructor(url, options)/options}}'s
    {{WebTransportOptions/serverCertificateFingerprints}} if it exists, and null otherwise.
-1. If |dedicated| is false and |serverCertificateFingerprints| is non-null, then [=throw=] a
-   {{TypeError}}.
+1. If |dedicated| is false and |serverCertificateFingerprints| is non-null, then [=throw=]
+   the result of [=WebTransportError/creating=] a {{WebTransportError}} with `"session"`.
```

Comment:
I feel like this one should stay a TypeError, since it refers to the parsing of the arguments rather than any property of the session, and it happens synchronously during construction.

Reply:
Makes sense, done.